bcgov/sbc-search
# Fixed vuetify issue.

name: PullRequest
description: Pull Request
title: "[issue]: "
body:
  - type: markdown
    attributes:
      value: |
        Thanks for submit the code!
  - type: dropdown
    id: purpose
    attributes:
      label: purpose
      description: What kinds of change?
      options:
        - New Feature
        - Enhancement
        - Bug
        - Devops
    validations:
      required: true        
  - type: input
    id: ticket
    attributes:
      label: Ticket Number:
      description: What's the ticket number
      placeholder: https://github.com/bcgov/entity/issues/
    validations:
      required: false
  - type: textarea
    id: description
    attributes:
      label: Description of changes:
      description: Also tell us, what did you expect to happen?
      placeholder: Tell us what you see!
      value: "A bug happened!"
    validations:
      required: true
